integration/apps/rails-seven/config/initializers/datadog.rb

```diff
@@ -11,7 +11,7 @@
   if Datadog::DemoEnv.feature?('tracing')
     c.tracing.analytics.enabled = true if Datadog::DemoEnv.feature?('analytics')
 
-    c.tracing.instrument :rails, request_queuing: true
+    c.tracing.instrument :rails, request_queuing: :exclude_request
```

Comment:
I wonder if this should be the default for the Rails integration app? Somewhat indifferent.

Need to think more about what these settings mean to an integration app: its not really meant to test all the different flags, as much as it is meant to test holistic stability of Rails 7 running typical `ddtrace` deployments.

Reply:
I'll happily defer to you here @delner, I did a grep for `request_queuing` to make sure I didn't miss anything, but will adjust as you see fit.